packages/chevrotain/api.d.ts

```diff
@@ -891,11 +891,21 @@ export declare class Parser {
      * @returns - The parsing rule which is the production implementation wrapped with the parsing logic that handles
      *                     Parser state / error recovery&reporting/ ...
      */
+    /* protected */ RULE(
+        name: string,
+        implementation: (...implArgs: any[]) => void,
+        config?: IRuleConfig<any>
+    ): (idxInCallingRule?: number, ...args: any[]) => CstNode
+
+    /**
+     * @see RULE
+     * @hidden
```

Comment:
What is "@hidden" annotation for?

Reply:
I'm following your earlier suggestion, as I'm not sure which documentation generator this project is using, or how it is configured. Will remove it.

Reply:
Good point about doc generator, I will test the generated docs before merging this.